_data/changelogs/component-identifier.yml

```diff
@@ -2,6 +2,12 @@ title: Identifier
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added clarification on parent agency.
+    summaryAdditional: “Parent agency” means "the highest-level agency associated with a site or service." We added guidance to cover cases when a site is the primary site for an agency, and has no higher-level agency.
```

Comment:
No comma is needed between “agency” and “and.” If you want to simplify the sentence I think it would be ok to say, “We added guidance to cover cases where a site has no higher-level agency.”

Reply:
```suggestion
    summaryAdditional: We added guidance to cover cases where a site has no higher-level agency.
```